src/main/java/ch/wisv/payments/admin/products/ProductServiceImpl.java

```diff
@@ -130,13 +142,32 @@ public boolean isProductAvailable(Integer productId) {
     }
 
     @Override
-    public void deleteProduct(int productId) {
-        List<Order> orders = orderService.getOrdersByProductId(productId);
+    public ProductGroup getProductGroupById(Integer productGroupId) {
+        return productGroupRepository.findOne(productGroupId);
+    }
 
-        if (orders.size() > 0) {
-            throw new ProductInUseException("Products are already ordered");
+    @Override
+    public void editProductGroup(ProductGroupRequest productGroupRequest) {
+        if (productGroupRequest.getId() != 0) {
```

Comment:
It was also in the [`editProduct()` method](https://github.com/WISVCH/payments/pull/40/files#diff-94153d1e54f3ee6886d4c10e00affbd5R83), could you shed some light on this?